Podcast/SearchPeopleTableViewCell.swift

```diff
@@ -99,22 +98,23 @@ class SearchPeopleTableViewCell: UITableViewCell {
         profilePictureImageView.setImageAsynchronouslyWithDefaultImage(url: user.imageURL, defaultImage: #imageLiteral(resourceName: "person"))
         profilePictureImageView.sizeToFit()
         nameLabel.text = user.firstName + " " + user.lastName
-        setFollowButtonState(isFollowing: user.isFollowing)
         if user.id == System.currentUser?.id {
             followButton.isHidden = true 
         }
-        detailLabel.text = "@\(user.username) • \(user.numberOfFollowers.shortString()) followers"
+        username  = user.username
+        setFollowButtonState(isFollowing: user.isFollowing, numberOfFollowers: user.numberOfFollowers)
```

Comment:
![image](https://user-images.githubusercontent.com/20246620/32375795-411faa86-c079-11e7-9079-8ce6f8f16d9a.png)
![image](https://user-images.githubusercontent.com/20246620/32375797-45255734-c079-11e7-8ffc-30fa55a52b38.png)
I followed you before searching you but it's not updated in the search results? This might be a backend issue

Reply:
Backend issue!